.github/workflows/frontend-shared.yml

```diff
@@ -0,0 +1,34 @@
+name: frontend-shared package
+on:
+  release:
```

Comment:
To check my understanding - the frontend-shared package gets updated when there is a _client release_? That will might work OK since we ship new client releases all the time. However it is also likely to mean there are a lot of releases of this package with no changes in them. Perhaps this could bail out early if there are no changes in `frontend-shared/` since the previous release?

Reply:
Indeed. Off the top of my head, something like this could work.

`npm view hypothesis version `  -- assuming thats the version before the client that was just published. We may have subtract one depending on order.

Then, use that number and run this command.

`git diff v<that version #>..master frontend-shared/`

and if the stdout is anything but null/empty, then we should do a release.